cli/tokio_util.rs

```diff
@@ -29,66 +28,3 @@ where
 {
   tokio::runtime::current_thread::run(future.boxed().compat());
 }
-
-/// THIS IS A HACK AND SHOULD BE AVOIDED.
-///
-/// This spawns a new thread and creates a single-threaded tokio runtime on that thread,
-/// to execute the given future.
-///
-/// This is useful when we want to block the main runtime to
-/// resolve a future without worrying that we'll use up all the threads in the
-/// main runtime.
-pub fn block_on<F, R>(future: F) -> Result<R, ErrBox>
```

Comment:
Very happy to see this go away. I think we still haven't really solved the core problem - but much better to be using `futures::executor::block_on` rather than this home-brewed garbage. Nice work!